genai-perf/genai_perf/export_data/telemetry_data_exporter_util.py

```diff
@@ -0,0 +1,190 @@
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from typing import Dict, List, Optional
+
+from rich.console import Console
+from rich.table import Table
+
+
+class TelemetryDataExporterUtil:
+
+    TELEMETRY_DYNAMIC_METRICS_HEADER = [
+        "Metric",
+        "GPU",
+        "avg",
+        "min",
+        "max",
+        "p99",
+        "p95",
+        "p90",
+        "p75",
+        "p50",
+        "p25",
+    ]
+
+    TELEMETRY_CONSTANT_METRICS_HEADER = [
+        "Metric",
+        "GPU",
+        "Value",
+    ]
+
+    TELEMETRY_CONSTANT_METRICS = ["gpu_power_limit", "total_gpu_memory"]
+
+    TELEMETRY_GROUPS = {
+        "Power": ["gpu_power_usage", "gpu_power_limit", "energy_consumption"],
+        "Memory": ["gpu_memory_used", "total_gpu_memory"],
+        "Utilization": ["gpu_utilization"],
+    }
+
+    @staticmethod
+    def merge_telemetry_stats_json(
+        telemetry_stats: Optional[Dict], stats_and_args: Dict
+    ) -> None:
+        if telemetry_stats is not None:
+            stats_and_args.update({"telemetry_stats": telemetry_stats})
+
+    @staticmethod
+    def export_telemetry_stats_csv(telemetry_stats: Optional[Dict], csv_writer) -> None:
+        if telemetry_stats:
+            TelemetryDataExporterUtil._write_dynamic_telemetry_stats(
+                telemetry_stats, csv_writer
+            )
+            TelemetryDataExporterUtil._write_constant_telemetry_stats(
+                telemetry_stats, csv_writer
+            )
+
+    @staticmethod
+    def export_telemetry_stats_console(
+        telemetry_stats: Optional[Dict], STAT_COLUMN_KEYS: List[str], console: Console
```

Comment:
Is this supposed to be capitalized? I think anywhere you are using something as a parameter rather than a constant, it should be lower snake case to adhere to PEP8. Could you go through the changes to do so, if you think that makes sense?